Extensions/NotificationService/NotificationService.swift

```diff
@@ -77,13 +80,17 @@ class SyncDataDisplay {
 
     func displayNotification(_ message: PushMessage? = nil, with error: PushMessageError? = nil) {
         guard let message = message, error == nil else {
+            if let error = error {
+                SentryIntegration.shared.send(message: "PushMessageError: \(error.description)", tag: sentryTag)
```

Comment:
should we include the stacktrace here too since we're catching an error?

Reply:
Perhaps. The very next thing we do here is display an unknown message, which causes us to send a stack trace.